File: abc/abc280/c.py
# https://atcoder.jp/contests/abc280/submissions/37403779

s = input()
t = input()

ans = min(len(s), len(t)) + 1
for i in range(min(len(s), len(t))):
  if s[i] != t[i]:
    ans = i + 1
    break

print(ans)

File: abc/abc280/c.cpp
// https://atcoder.jp/contests/abc280/submissions/37006921

#include <bits/stdc++.h>
using namespace std;

int main() {
  cin.tie(0);
  ios_base::sync_with_stdio(0);

  string s, t;
  cin >> s >> t;

  for(int i = 0; i < s.size(); i++) {
    if(s[i] != t[i]) {
      cout << i + 1;
      return 0;
    }
  }

  cout << t.size();
  return 0;
}
